packages/edit-post/src/components/header/index.js

```diff
@@ -94,7 +94,7 @@ function Header() {
 					aria-expanded={ isEditorSidebarOpened }
 					shortcut={ shortcut }
 				/>
-				<PinnedPlugins.Slot />
+				<PinnedItems.Slot scope="edit-post/complementary-area" />
```

Comment:
It should list all types of pinned items for a given page, it was never meant to work only with sidebar plugins:

```suggestion
				<PinnedItems.Slot scope="edit-post" />
```